app/screens/thread/thread_base.js

```diff
@@ -88,7 +88,8 @@ export default class ThreadBase extends PureComponent {
     }
 
     close = () => {
-        this.props.actions.popTopScreen();
+        const {actions, componentId} = this.props;
+        actions.popTopScreen(componentId);
```

Comment:
I'm kind of 50/50 on whether or not we should just call `Navigation.pop` instead of `actions.popTopScreen` here, but I do like the abstraction that we get by calling the action still